pysteps/tests/test_exporters.py

```diff
@@ -14,7 +23,36 @@ def test_get_geotiff_filename():
     timestep = 5
 
     for timestep_index in range(n_timesteps):
-        file_name = _get_geotiff_filename("test/path", start_date, n_timesteps, timestep,
+        file_name = _get_geotiff_filename("test/path",
+                                          start_date, n_timesteps, timestep,
                                           timestep_index)
-
-        assert f"test/path_201909082022_{(timestep_index + 1) * timestep:03d}.tif" == file_name
+        expected = (f"test/path_201909082022_"
+                    f"{(timestep_index + 1) * timestep:03d}.tif")
+        assert expected == file_name
+
+
+def test_io_export_netcdf_one_member_one_time_step():
+    """Test the export netcdf."""
+    # open a netcdf file
+    root_path = pysteps.rcparams.data_sources["bom"]["root_path"]
+    rel_path = os.path.join("prcp-cscn", "2", "2018", "06", "16")
+    filename = os.path.join(root_path, rel_path,
+                            "2_20180616_100000.prcp-cscn.nc")
+    precip, _, metadata = pysteps.io.import_bom_rf3(filename)
+
+    # save it back to disk
+    with tempfile.TemporaryDirectory() as outpath:
+        outfnprefix = 'test_netcdf_out'
+        file_path = os.path.join(outpath, outfnprefix+'.nc')
+        startdate = pd.to_datetime('2018-06-16T10:00:00')
```

Comment:
can this be done with datetime instead of pandas to avoid having that dependency?

`startdate = datetime.strptime("2018-06-16 10:00:00", "%Y-%m-%d %H:%M:%S")`

Reply:
Yes. It makes sense. I have just pushed a new version without pandas as per your suggestion.